docs/toolbox.md

```diff
@@ -0,0 +1,48 @@
+<!--[metadata]>
++++
+title = "Toolbox Overview"
+description = "Documentation that provides an overview of Toolbox"
+keywords = ["docker, documentation, about, technology, kitematic, gui, toolbox"]
+[menu.main]
+parent="workw_toolbox"
+weight=1
++++
+<![end-metadata]-->
+
+# Docker Toolbox Overview
+
+Docker Toolbox is an installer for quick setup and launch of a Docker environment on Mac and Windows systems.
+
+![Toolbox installer](images/toolbox-installer.png)
+
+## What's in the box
+
+Toolbox includes these Docker tools:
+
+* Docker Machine for running `docker-machine` commands
+
+* Docker Engine for running the `docker` commands
+
+* Docker Compose for running the `docker-compose` commands
+
+* Kitematic, the Docker GUI
+
+* a shell preconfigured for a Docker command-line environment
+
+* Oracle VirtualBox
+
+## Ready to get started?
+
+<a href="https://www.docker.com/products/docker-toolbox" target="_blank">Download Docker Toolbox</a> from the product page.
+
+## Next Steps
+
+* <a href="https://docs.docker.com/mac/" target="_blank">Get Started with Docker for Mac OS X</a>
```

Comment:
should these links really be annoying popups?

and should they be to markdown files.